src/ex4/tasks.txt

```diff
@@ -0,0 +1 @@
+Catch snorlax
```

Comment:
add this file to `.gitgnore` because it should not be persistent to the git repo (anyone should have his own tasks file on the computer)